rules-topics/15_smithing.md

```diff
@@ -0,0 +1,99 @@
+# Smithing
+
+## Smithing Products
+
+### Tier I
+#### Alarm Trap
+   >This creates a trap that can be placed next to a doorway, or the opening to a box. The rune, once activated, will make a loud alarming sound. The trap must have a trigger and a tripwire, or equivalent that will set off the trigger. Alarm Traps may be disarmed as they are a tier I trap. Activating an Alarm Trap requires simply placing the trap rep, the trigger, the trip wire, and the character says "Activate Alarm Trap". This item is consumed on use.
+#### Key
+   >This recipe allows the Smith to create a key that is keyed for a particular lock by an ID code that both the key and the lock share. Keys will only work for the lock that they are keyed for. To make a Key for a lock, you must bring the lock to Logistics.
+#### Lock
+   >This recipe allows the Smith to create a lock able to keep out unskilled burglars. Lock physical representations are attached to the main door of a structure. This lock indicates that all portals are locked. The tag for the lock will be attached to the back of the lock presentation, and will note the quality of the lock as well as the lock ID code. To be able to enter a locked structure, a character must have a key tag that is ID associated with the lock. A character with a Key may unlock a Locked door on a 3 count, and may re-lock it on another 3 count. The rating of this lock is 1. This has 5 Durability.
+#### Rusty Weapon
+>The sign of a poor smith, or a genius, depending on who you ask. This process intentionally rusts the weapon to make it inflict foul wounds upon those it strikes. This process changes the base damage type of the weapon to Poison, at the cost of reducing it's Durability by 2. A Rusty Weapon cannot be Silvered.
+
+
+### Tier II
+
+#### Armor Resizing Kit
+> This kit has everything a blacksmith needs to adjust suits of armor to better suit their needs. A Smith using this kit may increase an existing suit of armor's value rating by spending additional resources. This does not change the durability of the armor. This has 5 charges.
```

Comment:
Can we add detail here about what the ratio is for materials needed per x armor points added?  

I might also suggest noting that increasing the armor points on the item doesn't change the amount of armor a character can actually use.